website/docs/reference/datasources/airtable.md

```diff
@@ -4,399 +4,334 @@ sidebar_position: 1
-* **Offset:** The response will contain an offset if there are more records. Pagination will stop when you’ve reached the end of your table.
+Record data is returned according to the format:
 
-<VideoEmbed host="youtube" videoId="-1NwawxZ-XI" title="Using the List Records Command" caption="Using the List Records Command"/>
-
-* Click on the **+** icon next to the **queries/js** and choose your airtable datasource.
-* Rename the query.
-* From the Commands drop-down, Select the method **`List Records`**.
-* Next, add your **Base ID** and **Table Name**.
-
-Once the parameters have been specified, click **Run**. The response section will now display all of the airtable data.
-
-To better understand how the list record command works, let's look at an _**example**._
-
-Let’s assume we have a task manager database, which displays - _Task Name, Deadline, and Task Manager’s Name,_ as shown below:
-
-```
-| Name                    | Deadline     | Manager |
-|-------------------------|--------------|---------|
-| Review OSW              | 25 June 2022 | Harshil |
-| Finalize budget         | 4 July 2022  | Pranay  |
-| Firestore Docs Update   | 30 June 2022 | Harshil |
-| Heroku Deployment Issue | 26 June 2022 | Nikhila |
-```
-
-<VideoEmbed host="youtube" videoId="YKOFXEQUtgc" title="List Record Example" caption="List Record Example"/>
-
-* After creating the **list\_records** query, Click on the plus **(+)** icon next to the queries/js and create a New **JS Object.**
-* Rename the JS Object to **GetAirtableRecordsData**.
-* Now, paste the below code:
-
-```
-export default {
-	readAirtableData: () => {
-		return list_tasks.data.records.map((airtableRecord) => {
-			return {
-				"id" : airtableRecord.id,
-				"createdTimeDate" : airtableRecord.createdTime,
-				"Manager" : airtableRecord.fields.Manager,
-				"TaskName" : airtableRecord.fields.Name,
-				"Deadline" : airtableRecord.fields.Deadline
-			}
-		})
-	}
+```json
+{
+  "records": [
+    {
+      "id": "rec2BjrZNxNjhJ9dO",
+      "createdTime": "2023-03-13T15:11:26.000Z",
+      "fields": {
+        "name": "Bob Jones",
+        "employee_id": 1001,
+        // ...
+      }
+    },
+    // further records...
+  ],
+  "offset": "itr534r2Ro2QnjK3x/rec2BjrZNxNjhJ9dO"
 }
 ```
 
-* Next, add a table widget to your canvas, and name it **`Tasks_list`**.
-* In the Table data property, paste:
+To access your record fields in a widget, it's helpful to use a `map()` function:
 
-```
-{{GetAirtableRecordsData.readAirtableData()}}
+```javascript
+// binding response data to a Table widget
+{{
+  AirtableQuery.data.records.map(row => {
+    return {
+      "Name": row.fields.name,
+      "Employee ID": row.fields.employee_id
+    }
+  })
+}}
 ```
 
-The [table widget](./../widgets/table/README.md) will display all of your Airtable data.
+### Filter and sort
```

Comment:
I should there should be a table above this listing all the fields in the query and below you can elaborate on Filter, sort and serverside pagination